data/json/start_locations.json

```
@@ -203,7 +203,7 @@
     "type": "start_location",
     "id": "sloc_gun_store",
     "name": "Gun Store",
-    "terrain": [ "s_gun", "s_gun_1", "s_gun_2", "s_gun_3", "s_gun_4" ]
+    "terrain": [ "s_gun_a", "s_gun_b" ]
```

```suggestion
    "terrain": [ "s_gun_a", "s_gun_b_store" ]
```
Test is failing because there is no s_gun_b. (Terrain is either sun_gun_b_store or s_gun_b_range)

---

Ah, thank you! I had assumed it used the overmap id, not the mapgen id